lib/curses_renderer.rb

```diff
@@ -57,6 +45,16 @@ def render
     @screen.refresh
   end
 
+  def take_player_turn
+    key = @screen.getch
+    if key == 'q' || key == 'Q'
+      quit!
+    elsif movement?(key)
+      update_selection(key)
+    elsif painting?(key)
+      @board.paint!(@selected[0], @selected[1], :player)
```

Comment:
Here we communicate who did the painting!

Now it's a little bit cumbersome in arguments, selected row, column, and who. One of the things I was building toward was making our row/column code a little more concise and we could select both at once like:

```ruby
@board.paint!(@selected, :player)
```

`@selected` could stay an array or it could become a custom `BoardIndex` class that tracks its own `@row` and `@col`.

Reply:
So, we consolidating @selected[0] and @selected[1] into a single parameter, @selected, to help make code easier to read? Maybe a separate method in the rendering file? I'm not totally sure what that would look like as a BoardIndex class, is this what you mean? 

```
class BoardIndex
  attr_accessor :row, :col
  def initialize(row, col)
    @row = row
    @col = col
  end
end
```

```
@selected = BoardIndex.new(2, 3)
```